docker-compose-deploy.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.7"
+
```

Comment:
I would separate the 2 docker-compose files into 2 separate directories (so one can cd into the appropriate directory and run `docker-compose up -d`
or include the info that in order to run "docker-compose-deploy.yaml" one must enter 
`docker-compose -f docker-compose-deploy.yaml up`